File: 课程/muisdflkjkasdf/STApp.m
//
// Created by kemchenj on 5/30/16.
// Copyright (c) 2016 kemchenj. All rights reserved.
//

#import "STApp.h"



@implementation STApp

- (STApp *)appWithDict:(NSDictionary *)dictionary;
{
    STApp *app = [[STApp alloc] init];
    [app setValuesForKeysWithDictionary:dictionary];
    return app;
}


@end

File: 课程/multiPicDownload/multiPicDownload/STApp.swift
//
//  STApp.swift
//  multiPicDownload
//
//  Created by kemchenj on 5/30/16.
//  Copyright © 2016 kemchenj. All rights reserved.
//

import UIKit

class STApp: NSObject {
    
    var name : String?
    var icon : String?
    var download : String?
    
    init(dict: Dictionary<String, String>) {
        super.init()
        self.setValuesForKeysWithDictionary(dict)
    }

}
